SmartCarApplication/app/src/main/java/com/example/smartcarapplication/MainActivity.java

```diff
@@ -116,8 +117,7 @@ public void onSuccess(IMqttToken asyncActionToken) {
                     final String successfulConnection = "Connected to MQTT broker";
                     Log.i(TAG, successfulConnection);
                     Toast.makeText(getApplicationContext(), successfulConnection, Toast.LENGTH_SHORT).show();
-                    mMqttClient.subscribe("/smartcar/ultrasound/front", QOS, null);
-                    mMqttClient.subscribe("/smartcar/camera", QOS, null);
+                    mMqttClient.subscribe("/smartcar/#", QOS, null);
```

Comment:
This is good. I was wondering how instead of writing all individual subscribe() methods in setup, there might be a way to subscribe to all (frontcam, throttle, steering, and more) at once. Great hack!